cypress-tests/cypress/support/commands.js

```diff
@@ -2365,26 +2310,38 @@ Cypress.Commands.add(
   }
 );
 
-Cypress.Commands.add("stepUp", (stepUpBody, globalState, value) => {
-  const key = `step_up_enabled_${globalState.get("merchantId")}`;
-  stepUpBody.key = key;
-  stepUpBody.value = value;
+Cypress.Commands.add("updateConfig", (configType, configData, globalState, value) => {
+  let url;
+  let body;
+  
+  switch (configType) {
+    case 'autoRetry':
+      url = `${globalState.get("baseUrl")}/configs/autoRetry`;
+      body = { ...configData, enable: value }; // enable or disable autoRetry
+      break;
+    case 'maxRetries':
+      url = `${globalState.get("baseUrl")}/configs/maxRetries`;
+      body = { ...configData, retries: value }; // set max retries
+      break;
+    case 'stepUp':
+      url = `${globalState.get("baseUrl")}/configs/stepUp`;
+      body = { ...configData, connectors: value }; // step-up configurations
+      break;
+    default:
+      throw new Error('Invalid config type');
```

Comment:
Can we have clear error message? This needs to be done just so that, if tomorrow, when Cypress throws this error, we should be able to see what config type was passed:
```suggestion
      throw new Error(`Invalid config type passed into the configs: "${key}: ${value}"`);
```